esti/migrate_test.go

```diff
@@ -121,20 +138,86 @@ func createAndUploadParts(t *testing.T, logger logging.Logger, resp *s3.CreateMu
 func testPostMigrateMultipart(t *testing.T) {
 	ctx := context.Background()
 
-	partsConcat, completedParts := createAndUploadParts(t, logger, &state.Multipart.Info, migratePostPartsCount, migratePrePartsCount)
+	partsConcat, completedParts := createAndUploadParts(t, logger, &state.Multiparts.Info, migratePostPartsCount, migratePrePartsCount)
 
-	completeResponse, err := uploadMultipartComplete(svc, &state.Multipart.Info, append(state.Multipart.CompletedParts, completedParts...))
+	completeResponse, err := uploadMultipartComplete(svc, &state.Multiparts.Info, append(state.Multiparts.CompletedParts, completedParts...))
 	require.NoError(t, err, "failed to complete multipart upload")
 
 	logger.WithField("key", completeResponse.Key).Info("Completed multipart request successfully")
 
-	getResp, err := client.GetObjectWithResponse(ctx, state.Multipart.Repo, mainBranch, &api.GetObjectParams{Path: migrateMultipartFile})
+	getResp, err := client.GetObjectWithResponse(ctx, state.Multiparts.Repo, mainBranch, &api.GetObjectParams{Path: migrateMultipartsFile})
 	require.NoError(t, err, "failed to get object")
 	require.Equal(t, http.StatusOK, getResp.StatusCode())
 
-	preContentBytes, err := base64.StdEncoding.DecodeString(state.Multipart.Content)
+	preContentBytes, err := base64.StdEncoding.DecodeString(state.Multiparts.Content)
 	require.NoError(t, err, "failed to decode error")
 
 	fullContent := append(preContentBytes, partsConcat...)
 	require.Equal(t, fullContent, getResp.Body)
 }
+
+// Actions Tests
+
+func testPreMigrateActions(t *testing.T) {
+	// Create action data before migration
+	ctx, _, repo := setupTest(t)
+	state.Actions.Repo = repo
```

Comment:
I know it's annoying but since `Repo` is being handled differently in the migration struct and path, I think you should test more than a single repo here.

Reply:
I guess we can add it to this test - it will be quite a headache. Trying to understand what is the benefit of adding it to this test specifically? What do we want to check here?

Reply:
As discussed, added this to the actions migrate unit test

Reply:
Couldn't find it 🤷

Reply:
TestMigrate in pkg/actions/kv_migrate_test.go